app/lib/screens/wallets/wallet_info.dart

```diff
@@ -28,11 +28,15 @@ class _WalletDetailsWidgetState extends ConsumerState<WalletDetailsWidget> {
   bool showStellarSecret = false;
   bool edit = false;
   late WalletsNotifier walletsRef;
+  String? _errorText;
 
   @override
   void initState() {
     super.initState();
     walletsRef = ref.read(walletsNotifier.notifier);
+    walletNameController.text = widget.wallet.name;
+    walletName = widget.wallet.name;
+    walletNameController.addListener(_validateWalletName);

```

Comment:
why do we use listeners? can't we use `onChanged` on the `textfield`?

Reply:
fixed